src/_includes/layouts/base.njk

```diff
@@ -119,6 +119,7 @@
                             <a class="flex items-center gap-2">resources {% include "components/icons/chevron-down.svg" %}</a>
                             <ul>
                                 <li><a href="/blog">blog</a></li>
+                                <li><a href="/community/newsletter">newsletter</a></li>
```

Comment:
Just curious, why no capital letters for the link text?

Reply:
Our logo is all lowercase, and when originally designing website, it looked strange to have any capital letters, so our navigation had just stayed all lower case